src/Components/Error/Error.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import './Error.css'
 
-const Error = () => {
-
-}
+const Error = () => (
```

Comment:
I think `<Error />` should take in some kind of error object, and dynamically display information from that object.

Reply:
At the very least we should display an error status code if we have one. Oftentimes errors will include a message we can display as well.

Reply:
Sure! And we'll be potentially passing in that object wherever the `Error` is getting displayed?

Reply:
What specifically would you like for a message outside of the generic `Something went wrong, please try again later`?

Reply:
Re first question: Yes. On previous projects I've included an `error` property in `<App />` and then something like `{this.state.error && <Error error={this.state.error} />}`. I'm flexible about how that structure can work in this project, especially considering we're using hooks. 

Re second question: For now let's assume that `error.status` will return a status code and `error.statusText` will return a message describing the error. These are default properties we get when we use the `throw()` method. Depending on specifics this could change.